src/components/FigureCard/FigureCard.vue

```diff
@@ -1,12 +1,16 @@
 <template>
   <figure class="figure-card">
-    <div class="image-wrapper">
+    <figcaption v-show="isTitleUp" v-if="title" :data-test-id="`${dataTestId}-title`">
```

Comment:
Please remove the `v-show` attribute in favor of updating `v-if`:

```js
<figcaption v-if="titleAbove && title" :data-test-id="`${dataTestId}-title`">
```

Reply:
Nice! Thanks for the recommendation